clndr/index.d.ts

```diff
@@ -0,0 +1,383 @@
+// Type definitions for clndr 1.4
+// Project: https://github.com/kylestetz/CLNDR
+// Definitions by: jasperjn <https://github.com/jasperjn>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="jquery" />
+import * as moment from 'moment';
+
+declare namespace Clndr {
```

Comment:
This namespace won't be visible from the outside. Why not just export these interfaces from the module?